packages/twenty-front/src/modules/object-record/record-merge/hooks/useMergePreview.ts

```diff
@@ -6,6 +6,7 @@ import { useMergeManyRecords } from '@/object-record/hooks/useMergeManyRecords';
 import { useMergeRecordRelationships } from '@/object-record/record-merge/hooks/useMergeRecordRelationships';
 import { useUpsertRecordsInStore } from '@/object-record/record-store/hooks/useUpsertRecordsInStore';
 import { type ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { mergeInProgressState } from '../states/mergeInProgressState';
```

Comment:
Nit: would you mind renaming to isMergeInProgress? It's the pattern most frequently used